cmd/oras/cp.go

```diff
@@ -52,54 +49,62 @@ func copyCmd() *cobra.Command {
 
 ** This command is in preview and under development. **
 
-Example - Copy the artifact tagged with 'v1' from repository 'localhost:5000/net-monitor' to repository 'localhost:5000/net-monitor-copy' 
-  oras cp localhost:5000/net-monitor:v1 localhost:5000/net-monitor-copy:v1
+Example - Copy an artifact between registries:
+  oras cp localhost:5000/net-monitor:v1 localhost:6000/net-monitor-copy:v1
 
-Example - Copy the artifact tagged with 'v1' and its referrers from repository 'localhost:5000/net-monitor' to 'localhost:5000/net-monitor-copy'
-  oras cp -r localhost:5000/net-monitor:v1 localhost:5000/net-monitor-copy:v1
+Example - Download an artifact into an OCI layout folder:
+  oras cp --to-oci localhost:5000/net-monitor:v1 ./downloaded:v1
```

Comment:
To reduce ambiguity, we can rename `--to-oci` to `--to-oci-layout`, and same for `--to-oci`.